docs/source/docker-compose.rst

```diff
@@ -1,48 +1,28 @@
+.. _Using Docker Compose:
 Using Docker Compose
 ===================================
 
-To automate DAG executions based on cron expressions, it is necessary to run both the ui server and scheduler process. Here is an example `docker-compose.yml` setup for running Dagu using Docker Compose.
+Here is an example `docker-compose.yml` setup for running Dagu using Docker Compose.
 
 .. code-block:: yaml
 
-  version: "3.9"
-  services:
-      # init container updates permission
-      init:
-          image: "ghcr.io/dagu-org/dagu:latest"
-          user: root
-          volumes:
-              - dagu_config:/home/dagu/.config/dagu
-              - dagu_data:/home/dagu/.local/share
-          command: chown -R dagu /home/dagu/.config/dagu/ /home/dagu/.local/share
-      # ui server process
-      server:
-          image: "ghcr.io/dagu-org/dagu:latest"
-          environment:
-              - DAGU_PORT=8080
-              - DAGU_TZ=Asia/Tokyo
-          restart: unless-stopped
-          ports:
-              - "8080:8080"
-          volumes:
-              - dagu_config:/home/dagu/.config/dagu
-              - dagu_data:/home/dagu/.local/share
-          command: dagu server
-          depends_on:
-              - init
-      # scheduler process
-      scheduler:
-          image: "ghcr.io/dagu-org/dagu:latest"
-          environment:
-              - DAGU_TZ=Asia/Tokyo
-          restart: unless-stopped
-          volumes:
-              - dagu_config:/home/dagu/.config/dagu
-              - dagu_data:/home/dagu/.local/share
-          command: dagu scheduler
-          depends_on:
-              - init
-  volumes:
-      dagu_config: {}
-      dagu_data: {}
+    services:
+        dagu:
+            image: "ghcr.io/dagu-org/dagu:latest"
+            container_name: dagu
+            hostname: dagu
+            ports:
+            - "8080:8080"
+            environment:
+            - DAGU_PORT=8080 # optional. default is 8080
+            - DAGU_TZ=Asia/Tokyo
+            - DAGU_BASE_PATH=/dagu # optional. default is /
+            - PUID=1000 # optional. default is 1000
+            - PGID=1000 # optional. default is 1000
+            - DOCKER_GID=999 # optional. default is -1 and it will be ignored
+            volumes:
+            - dagu_config:/config
+            - /var/run/docker.sock:/var/run/docker.sock # optional. required for docker in docker
```

Comment:
I wasn't sure if this was worth including here. It maybe the comment should mention the `socat` alternative in the docs?